assets/icons.tsx

```diff
@@ -156,6 +156,66 @@ export const MoonSolidIcon = () => {
     </svg>
   );
 };
+export const Passwordreveal = () => {
```

Comment:
I would like to suggest you to rename this icon from `Passwordreveal` to `PasswordRevealIcon`